docs/pelion.md

```diff
@@ -0,0 +1,34 @@
+# Pelion
+
+## Requirements
+
+kube-router documentation [General Requirements](https://www.kube-router.io/docs/user-guide/#requirements)
+
+Make sure ipset is installed
+```
+apt update && apt install -y ipset
```

Comment:
Isn't docker a pre-requisite for build with docker? Also in build without docker method don't we need to install Go?

Reply:
Updated the documentation to contain the requirements needed for building.